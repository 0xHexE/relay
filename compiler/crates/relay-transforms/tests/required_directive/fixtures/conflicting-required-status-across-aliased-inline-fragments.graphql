fragment Foo on Node {
    ... on MaybeNodeInterface @alias(as: "named_fragment") {
        name @required(action: NONE)
    }
    name
}