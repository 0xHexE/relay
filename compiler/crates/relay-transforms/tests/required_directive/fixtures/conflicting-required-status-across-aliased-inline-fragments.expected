==================================== INPUT ====================================
fragment Foo on Node {
    ... on MaybeNodeInterface @alias(as: "named_fragment") {
        name @required(action: NONE)
    }
    name
}
==================================== OUTPUT ===================================
fragment Foo on Node @__childrenCanBubbleNull {
  ... on MaybeNodeInterface @__FragmentAliasMetadata
  # FragmentAliasMetadata {
  #     alias: WithLocation {
  #         location: conflicting-required-status-across-aliased-inline-fragments.graphql:60:62,
  #         item: "named_fragment",
  #     },
  #     type_condition: Some(
  #         Interface(0),
  #     ),
  # }
   {
    name @required(action: NONE) @__RequiredMetadataDirective
    # RequiredMetadataDirective {
    #     action: None,
    #     path: "named_fragment.name",
    # }
    
  }
  name
}
